Backend/router/v1/report.js

```diff
@@ -8,7 +8,7 @@ module.exports = () => {
   });
 
   router.all('/', (req, res) => {
-    res.status(404).send("This is not normal route");
+    res.status(404).json({msg: "This is not normal route"});
```

Comment:
This is a very kind msg to a  hacker. Why don't you send a status code instead and let the client handles it? Do not specify what is wrong in this way.

Reply:
The application error code will be handled by another issue soon. ( #60 )